### ISSUE TITLE
Should recognize variables used with differing case

### ISSUE THREAD

**Issue by mpaluchowski** [cflint/CFLint#242]
The following code:

```coldfusion
function foo() {
  var bar = 1;
  return Bar;
}
```

will produce 2 issues:

1. Local variable bar is not used in function foo, consider removing it, on line `var bar = 1;`
1. Variable Bar is not a valid name. Please use CamelCase or underscores, on line `return Bar;`

Issue 1 is a false positive, because the variable _is_ used, albeit with a different case. Issue 2 is valid, but doesn't really point out the real problem, that the same variable is used with inconsistent case.

I suggest CFLint becomes a bit smarter about this and instead of these two issues reports one, something a long the lines of: "Variable Bar is used with inconsistent case. Please use the same case in all references as in the declaration."

**Comment by ryaneberly** [comment]
@mpaluchowski ,  UnusedLocalVarChecker is now case insensitive.  Thanks!

(The 2nd message about camel case is a valid checker, disable it if you don't want it)
